pyuvdata/aipy_extracts.py

```diff
@@ -419,7 +419,7 @@ def _rdhd(self, name):
             assert t == 'b'
 
             for t in itype:
-                v, o = _miread.hread(h, offset, t)
+                v, o = _miriad.hread(h, offset, t)
```

Comment:
This feels like it should have been causing errors, but I haven't seen anything I can link to this.

Reply:
On current master, we're evidently not covering the entire enclosing `else` block with tests: 

https://codecov.io/gh/RadioAstronomySoftwareGroup/pyuvdata/src/master/pyuvdata/aipy_extracts.py#L422

We should try to get better coverage for `aipy_extracts`, it's just not obvious to me how to best do that... Peter and I tried our best when we first moved it over and I think we've picked all the low-hanging fruit. Maybe we write some purposely-broken/odd test files?

Reply:
ah that explains it 🤦‍♀

Reply:
writing some purposely broken test files is a useful approach. Sometimes you can even generate them in the code, but if you can't we can write them out and save them in our test data. Well worth it to get the coverage up.